### DISCUSSION TITLE
1주차 과제 - 블로그 링크

### DISCUSSION THREAD

**Discussion by youngsangcho** [youngsangcho/generativedesign2_2021#6] [Show and tell]
1주차 과제

제너러티브 디자인 1 과제 중, 가장 마음에 드는 과제를 골라 
그 작업을 설명하는 포스트 올리기

제목 : 1주차 과제
내용 : 이미지 혹은 비디오, 코드

언제나 아래 형식에 맞춰 답글 달아주세요. (예시 참조)
[본인이름]
[링크]

기한 : 2021.09.13 밤10시

**Comment by youngsangcho** [comment]
조영상
http://youngcho.egloos.com/6629816

**Comment by heechhiker** [comment]
채경희
http://liuyeon95.egloos.com/6841863

**Comment by rkdwldms0815** [comment]
강지은
https://blog.naver.com/PostList.naver?blogId=nina0815&from=postList&categoryNo=6

**Comment by KimHyunJuhg** [comment]
김현정
https://blog.naver.com/PostList.naver?blogId=khj317888&from=postList&categoryNo=35

**Comment by jwejwepark** [comment]
박주혜
https://blog.naver.com/glqkfl0726/222500721727

**Comment by eunjinhong** [comment]
홍은진
https://blog.naver.com/tnekf1012/222502058430

**Comment by julia11060** [comment]
황주원
https://blog.naver.com/wndnjs11060/222502523088

**Comment by SongHyeJu** [comment]
송혜주
https://blog.naver.com/thd7451/222504332525

**Comment by imyeeunok** [comment]
김예은
https://blog.naver.com/010312kye/222504541323

**Comment by Joeun410** [comment]
김조은

https://blog.naver.com/PostView.naver?blogId=joanne3456&Redirect=View&logNo=222504579763&categoryNo=1&isAfterWrite=true&isMrblogPost=false&isHappyBeanLeverage=true&contentLength=16273

**Comment by Nick24680** [comment]
김윤영
http://yoon2468.egloos.com/6843499

**Comment by SeyounJung01** [comment]
정세윤
https://blog.naver.com/seyounjj2001/222499332054

**Comment by youngsangcho** [comment]
오늘 수업 줌 링크 정보입니다.
http://club.uos.ac.kr/lect/2021206637001
